docs/deploy.md

```diff
@@ -81,7 +81,7 @@ The bundled files are located in the root directory `/output/html`.
 
 ## Run
 
-1. According to your deploy environment, check the related configurations in `api/conf/conf.yaml`, modify those variables if needed.
+1. According to your deploy environment, check the related configurations in `output/conf/conf.yaml`, modify those variables if needed.
```

Comment:
Why? The output folder is generated only when we run the shell.

Reply:
The binary program we run is in the `output` directory, so the config file it reads is `output/conf/conf.yaml`

Reply:
But we don't havr that directory in source codes, and users can choose not to run the binary file in that folder.

Reply:
Then he should change the config file in the directory he run. After  run `api/build.sh`, the directory of the binary file is `output`, changing `api/conf/conf.yaml` will not take effect.

Reply:
or what's your suggestion ? @juzhiyuan

Reply:
1. Before building the binary file, we should update the config file under the `/api/conf/` directory, right?
2. After building successfully, we have the output directory.
3. When users want to update the config file without rebuilding the binary file, they must modify the file under the `/output` directory.

Reply:
According to your opinion, we should add some tips in the default config file, to let users know they can update the config only when the binary is generated.